Mathlib/Analysis/Calculus/Deriv/Slope.lean

```diff
@@ -25,23 +25,18 @@ derivative, slope
 -/
 
 
-universe u v w
+universe u v
 
-noncomputable section
-
-open Topology Filter TopologicalSpace
-open Filter Set
+open Topology Filter TopologicalSpace Set
```

Comment:
I'd be careful with those: I believe this is a porting artefact, where one of these two lines should be replaced by `open scoped` instead. Can you see which of the `open`s can be downgraded this way, please?